woocommerce/woocommerce
# Package Release: Handle invalid releases

### All Submissions:






### Changes proposed in this Pull Request:

Running the release script over all packages produces an error because some packages should not be released if they have no updates. Trying to do so will cause an error from NPM. This PR introduces a check for version numbers to make sure a bump has occurred before trying to publish.

### How to test the changes in this Pull Request:

1. Run the publish tool on a package that has not been first prepped. In other words, the version has number has not been bumped.
```
./tools/package-release/bin/dev publish @woocommerce/number --branch=fix/packages-release-errors
```
2. Confirm the publish action has been skipped.

### Other information:




### FOR PR REVIEWER ONLY:
